src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -324,64 +326,101 @@ private void AddLineBreaks(SourceText text, IList<string> lines)
         }
 
         private bool TryGenerateExteriorTriviaAfterEnter(
-            SyntaxTree syntaxTree,
-            SourceText text,
-            int position,
+            Document newDocument,
```

Comment:
:memo: I'm not a fan of the reordering of `newDocument` and `originalDocument` here compared to `InsertOnEnterTyped`. They should each use the same order.